### ISSUE TITLE
Add Pytorch and torchvision support?

### ISSUE THREAD

**Issue by stevewongv** [elazarcoh/simply-view-image-for-python-debugging#3]
Dear developer, 
Thank you for your great effort! Could you add pytorch tensor value support? Actually, it's really simple to convert tensor to image, just use
```python
import torchvision
torchvision.utils.save_image( tensor, '/tmp/XXX.png' ) # float tensor
```
This command will convert 4D tensor (N,C,H,W) to an image.

**Comment by elazarcoh** [comment]
Thanks for the interest!

How is a 4D tensor an image? Do you mean 3D? Otherwise, which image from the batch to show? all of them?

**Comment by stevewongv** [comment]
Yes, all of them. But you can limit this function to the 1 or 3 channels tensor(N,1 or 3,H,W).  Here is an example:
The shape of tensor is [4,1,H,W].
The code is:
```python 
torchvision.utils.save_image( tensor, '/tmp/XXX.png' , normalize=False,  pad_value=255, padding = 10)
```
The result is:
![mask_asso](https://user-images.githubusercontent.com/12782946/126328759-4442facf-a419-439a-a254-6d3b6e993a23.png)

**Comment by elazarcoh** [comment]
I made a POC for it. works well.

Can you please describe the required configurations you would like to have (e.g. is `padding` should be configurable).
also, the `pad_value`, should it depends on the datatype of the Tensor?

Thanks!

**Comment by stevewongv** [comment]
`padding` is the width of boundary between images. `pad_value` is the colour of the boundary (default is 0 -> black, 255 is white).

**Comment by stevewongv** [comment]
So I think 10, 255 are fine for most cases.

**Comment by elazarcoh** [comment]
implemented in: fa85496dfa83f70277d3ff13da8edb0abf448366

should be up by now. I'm closing the issue.

if you have any issues with it, feel free to re-open it.
